BM/avx512vbmi/vbmi_test.c

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Copyright (c) 2022 Intel Corporation.
+
+/*
+ * vbmi_test.c:  Vector Byte Manipulation Instructions
+ *
+ * Author: Pengfei, Xu <pengfei.xu@intel.com>
+ *        - Add parameter and usage to add new instruction next
+ *        - Check output binary result
+ *        - Check zmm1 output relation with zmm2 and zmm3
+ *        - Solved stack smashing and core dump issue after execute program
+ */
+
+/*****************************************************************************/
+
+#include <stdio.h>
+#include <stdlib.h>
+#define N 64
+
+typedef unsigned long long al __attribute__((aligned (64)));
```

Comment:
typedef unsigned long long al __attribute__((aligned (64)));  is necessary for avx512vbmi testing.  It's a warning "Don't define this type".  It could be ignored.  Thanks!

Reply:
Thanks Pengfei for clarifying.